packages/kbn-crypto/BUILD.bazel

```diff
@@ -38,6 +38,7 @@ TYPES_DEPS = [
   "@npm//@types/node",
   "@npm//@types/node-forge",
   "@npm//@types/testing-library__jest-dom",
+  "@npm//resize-observer-polyfill"
```

Comment:
This is a server-side package. It shouldn't depend on `resize-observer-polyfill`.

Reply:
@spalger Did you happen to open an issue for this?

Reply:
I didn't, remind me, this is because of the customizations to the `tsconfig.base.json` file right? IIRC I was surprised to find that we actually have a `tsconfig.browser.json` file, but I'm guessing if we add the right types there the file isn't being used by enough places?

Reply:
Correct on both accounts. I did try scoping to `tsconfig.browser.json` but almost all plugins extend `tsconfig.base.json` directly.

Reply:
@mshustov do you think it would make sense for the issue to be "migrate any ts project with browser code to `tsconfig.browser.json`?

Reply:
We also discussed that we'd be willing to maintain these types in EUI and not rely a third-party package. If that resolves some hesitation here, I can follow up in a future EUI release (prior to 7.14) to remove these type deps.

Reply:
@spalger I think so. As I can see, we already started separating compilation targets in the packages migrated to Bazel. 
https://github.com/elastic/kibana/blob/881d89fba7cc8b7f963fcd5230a60b1f23ac8259/packages/kbn-i18n/BUILD.bazel#L64-L79
@mistic Coul you confirm it's how we are going to compile sources in the new build toolchain? I can approve the PR, providing we are going to fix the problem later.